community/kernel-api/src/main/java/org/neo4j/internal/kernel/api/schema/SchemaDescriptor.java

```diff
@@ -34,8 +35,15 @@
  * how this is done in eg. LabelSchemaDescriptor, and the SchemaProcessor and SchemaComputer interfaces need to be
  * extended with methods taking the new concrete type as argument.
  */
-public interface SchemaDescriptor
+public interface SchemaDescriptor extends SchemaDescriptorSupplier
 {
+    boolean isAffected( long[] entityIds );
+
+    enum PropertySchemaType
+    {
+        SCHEMA_ALL_TOKENS,
+        NON_SCHEMA_ANY_TOKEN
```

Comment:
this should be renamed as well

Reply:
Not sure I agree, but it definitely should be documented, very good point.